### ISSUE TITLE
Review: 1.4.11 Non-text Contrast (AA)

### ISSUE THREAD

**Issue by qbalsdon** [GetEvinced/mcag#33]
Task: Edit [MCAG](https://github.com/GetEvinced/mcag/blob/main/src/app/data/mcag.json) section 1.4.11 considering the mobile research questions (to be completed)

---
The visual presentation of the following have a contrast ratio of at least 3:1 against adjacent color(s):

    User Interface Components: Visual information required to identify user interface components and states, except for inactive components or where the appearance of the component is determined by the user agent and not modified by the author;

    Graphical Objects: Parts of graphics required to understand the content, except when a particular presentation of graphics is essential to the information being conveyed.
---
Sufficient Techniques for Success Criterion 1.4.11

Note: Other techniques may also be sufficient if they meet the success criterion. See [Understanding Techniques.](https://www.w3.org/TR/2014/NOTE-UNDERSTANDING-WCAG20-20140916/understanding-techniques.html#understanding-techniques)
Situation A: Color is used to identify user interface components or used to identify user interface component states

    G195: Using an author-supplied, visible focus indicator
    [G174: Providing a control with a sufficient contrast ratio that allows users to switch to a presentation that uses sufficient contrast ](https://www.w3.org/WAI/WCAG22/Techniques/general/G174.html)

Situation B: Color is required to understand graphical content

    G207: Ensuring that a contrast ratio of 3:1 is provided for icons
    [G209: Provide sufficient contrast at the boundaries between adjoining colors ](https://www.w3.org/WAI/WCAG22/Techniques/general/G209.html)

Failures for Success Criterion 1.4.11

    F78: Failure of Success Criterion 2.4.7 due to styling element outlines and borders in a way that removes or renders non-visible the visual focus indicator
